irohad/main/application.hpp

```diff
@@ -113,6 +112,7 @@ class Irohad {
   size_t redis_port_;
   std::string pg_conn_;
   size_t torii_port_;
+  size_t internal_port_;
```

Comment:
The name is very abstract, better to add a description of the field.

Reply:
Added documentation to application.hpp